torch/utils/hipify/hipify_python.py

```diff
@@ -625,7 +618,7 @@ def is_out_of_place(rel_filepath):
     return True
 
 
-# Keep this synchronized with includes/ignores in build_amd.py
+# deprecated
```

Comment:
If this is to deprecate public APIs to be removed later, we should throw a warning for anyone using them

Reply:
Done.